drf_braces/fields/modified.py

```diff
@@ -24,6 +24,24 @@ def quantize(self, value):
         return super(DecimalField, self).quantize(value)
 
 
+class CurrencyField(DecimalField):
```

Comment:
can we rename to `RoundedDecimalField` since this field is not specifically currency related?